### ISSUE TITLE
TypeError: expected string or bytes-like object while checking for file existance using check_file_existence

### ISSUE THREAD

**Issue by jilju** [red-hat-storage/ocs-ci#1609]
Test cases which use check_file_existence function are intermittently failing with error:

```
pattern = '/var/lib/www/html/pod-test-rbd-1ec9836e93a742e891f12e1b34ce3c16'
string = None, flags = 0

    def search(pattern, string, flags=0):
        """Scan through string looking for a match to the pattern, returning
        a match object, or None if no match was found."""
>       return _compile(pattern, flags).search(string)
E       TypeError: expected string or bytes-like object

/usr/lib64/python3.6/re.py:182: TypeError
```

This occurs when stdout of find command is not obtained. 
Log:
```
2020-03-04 22:06:17,081 - INFO - ocs_ci.utility.utils.run_cmd.403 - Executing command: oc -n namespace-test-d0941e36e4854808adbe606f157e348a --kubeconfig cluster/auth/kubeconfig rsh pod-test-rbd-d144bb3608da4b509a0aca520a322075 bash -c "find /var/lib/www/html/pod-test-rbd-1ec9836e93a742e891f12e1b34ce3c16"
2020-03-04 22:06:18,247 - DEBUG - ocs_ci.utility.utils.run_cmd.414 - Command output: 
```

One of the failed test - [here](https://ceph-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/ocs-ci/376/testReport/junit/tests.manage.pv_services.test_dynamic_pvc_accessmodes_with_reclaim_policies/TestRWODynamicPvc/test_rwo_dynamic_pvc_CephBlockPool_Retain_/)

**Comment by jilju** [comment]
The error is not because file is absent. So adding retry will help to avoid test failure. #1612 
Retry can be removed when this issue is fixed.

**Comment by jilju** [comment]
Closing the old issue considering the fix #1612 as permanent.
